tests/api_resources/test_account.py

```diff
@@ -151,101 +71,116 @@ def test_verify_additional_owner(self):
                         },
                     },
                 },
-            },
-            None,
-        )
-
-    def test_login_link_create(self):
-        acct_id = 'acct_EXPRESS'
-        acct = stripe.Account.construct_from({
-            'id': acct_id,
-            'login_links': {
-                'object': 'list',
-                'data': {},
-                'has_more': False,
-                'url':  '/v1/accounts/' + acct_id + '/login_links'
             }
-        }, 'api_key')
+        )
+        self.assertIsInstance(resource, stripe.Account)
+        self.assertTrue(resource is account)
 
-        acct.login_links.create()
-        self.requestor_mock.request.assert_called_with(
+    def test_is_modifiable(self):
+        resource = stripe.Account.modify(
+            TEST_RESOURCE_ID,
+            metadata={'key': 'value'}
+        )
+        self.assert_requested(
             'post',
-            '/v1/accounts/' + acct_id + '/login_links',
-            {},
-            None
+            '/v1/accounts/%s' % TEST_RESOURCE_ID
         )
+        self.assertIsInstance(resource, stripe.Account)
 
+    def test_is_deletable(self):
+        resource = stripe.Account.retrieve(TEST_RESOURCE_ID)
+        resource.delete()
+        self.assert_requested(
+            'delete',
+            '/v1/accounts/%s' % resource.id
+        )
+        self.assertIsInstance(resource, stripe.Account)
 
-class AccountExternalAccountsTests(StripeResourceTest):
-    def test_create_external_account(self):
-        stripe.Account.create_external_account(
-            'acct_123',
-            source='btok_123'
+    def test_can_retrieve_no_id(self):
+        resource = stripe.Account.retrieve()
+        self.assert_requested(
+            'get',
+            '/v1/account'
         )
-        self.requestor_mock.request.assert_called_with(
+        self.assertIsInstance(resource, stripe.Account)
+
+    def test_can_reject(self):
+        account = stripe.Account.retrieve(TEST_RESOURCE_ID)
+        resource = account.reject(reason='fraud')
+        self.assert_requested(
             'post',
-            '/v1/accounts/acct_123/external_accounts',
-            {'source': 'btok_123'},
-            None
+            '/v1/accounts/%s/reject' % account.id
         )
+        self.assertIsInstance(resource, stripe.Account)
+        self.assertTrue(resource is account)
+
+    # TODO: Test deauthorize
```

Comment:
@ob-stripe Flagging that we need to test this. Did not fully grasp how to change the base on the stub and assert_requested

Reply:
Both `stub_request` and `assert_requested` will work regardless of the base URL.

This is because the base URL is set when constructing an `APIRequestor` instance, but our mocks will catch all calls to the `request` method on any `APIRequestor` instance.

I've pushed an update with a test for the `deauthorize` method.